R/derived-variable-functions.R

```diff
@@ -1,36 +1,86 @@
-# Custom ifelse for evaluating NA
+#' Custom ifelse function that evaluates missing (NA) values
+#' 
+#' If the logical argument compares to a value that is NA, it will set `x` to `FALSE`  
+#' 
+#' @param x A logical argument
+#' 
+#' @param a value if `x` is `TRUE`
+#' 
+#' @param b value if `x` is `FALSE`
+#' 
+#' @return None
+#' 
+#' @examples ifelse2((age<18),"child", "adult")
+#' 
+#' @export
 ifelse2 <- function(x, a, b) {
   falseifNA <- function(x) {
     ifelse(is.na(x), FALSE, x)
   }
   ifelse(falseifNA(x), a, b)
 }
 
-#BMI derived variable
-# HWTGHTM: height (in meters)
-# HWTGWTK: weight (in kilograms)
+#' Body Mass Index (BMI) derived variable
```

Comment:
The format looks good. Can you include:
-  a much expanded description of the problems with BMI variables in the individual example. 
- a recommendation when people would want to use this variable.
- other options.
- more info about the height and weight variables — that they need to first transform them and why.

Reply:
I wonder if we should include example of how to execute the function with Rec_W_Table()